tsconfig.json

```
@@ -5,8 +5,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "lib": ["es2020"],
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "module": "node16",
```

```suggestion
    "module": "node20",
```

we can just do the same as in CosmJS